lib/v2/screens/explore_screens/explore/interactor/viewmodels/explore_bloc.dart

```diff
@@ -1,8 +1,5 @@
 import 'package:bloc/bloc.dart';
 import 'package:seeds/v2/domain-shared/page_command.dart';
-import 'package:seeds/v2/domain-shared/page_state.dart';
```

Comment:
do we need to keep `ExploreStateMapper` or `GetExploreDataUseCase`?  If not, pls remove the files.

Reply:
I agree with Gery but we should remove even the isDHOmember call in the repository?

Reply:
Leave the repo call. 
Everything else thats not used. 🔥  it

Reply:
Although I would recommend to leave a comment above that call that says it is not used for the moment

Reply:
Done